physicalplan/plan.go

```diff
@@ -42,7 +42,7 @@ func newOperator(expr parser.Expr, storage storage.Queryable, mint time.Time, ma
 		return scan.NewNumberLiteralSelector(model.NewVectorPool(stepsBatch), mint, maxt, step, stepsBatch, e.Val), nil
 
 	case *parser.VectorSelector:
-		filter := scan.NewSeriesFilter(storage, mint, maxt, 0, lookbackDelta, e.LabelMatchers)
+		filter := scan.NewSeriesFilter(storage, mint, maxt, 0, lookbackDelta, e.OriginalOffset, e.LabelMatchers)
```

Comment:
Would everything still work if we just adjusted `mint` and `maxt` to avoid propagating this value in different operators?

Reply:
Good point, I think this works.